docker/Dockerfile

```diff
@@ -36,3 +36,4 @@ RUN apk add --no-cache --virtual .build-dependencies \
     apk del ${simulate_removal} .build-dependencies
 
 COPY search/ search/
+COPY search.json .
```

Comment:
configuration files should not be baked into an image, as it prevents reusing the image across environments; looking at the content this refers to a real environment (http://unstable--dummy-api.libero.pub) so the direction should be:
- run the `dummy-api` image as part of `docker-compose.yml` or `docker-compose.dev.yml` (only in the configurations where it's needed to perform real HTTP requests as part of tests or running the application)
- mount a `search.json` file pointing to that local container as part of `volumes`

Reply:
Agreed. I left this in as I wasn't sure about the correct approach. Further to our conversation in the office, I'll provide an empty config file as an example that will be added to the docker image by default, I'll add a dev config file in `/docker` that will be mounted to the dev container to run locally.